### DISCUSSION TITLE
Can I give you tree-sitter-nix repo ownership?

### DISCUSSION THREAD

**Discussion by cstrahan** [nix-community/infra#469] [Q&A] [answered]
A lot of editors (Helix, NeoVim, etc) are using my https://github.com/cstrahan/tree-sitter-nix grammar to provide syntax highlighting, indentation, folding, etc.

I don't want to be the bottleneck for bug fixes and innovation, so... would nix-community be interested in taking ownership of the repo? The idea would be that I could keep commit access so I can (at least try to) accept PRs and make any fixes (time permitting), but then several others could also be given commit access.

**Comment by cstrahan** [comment]
Someone asked recently asked if I could publish the Rust crate, so I think I'll do that tonight or tomorrow; I have similar thoughts there. Ideally, someone (@zimbatm?) would also be made a co-maintainer for the Rust crate, just in case I get hit by a bus or something, lol.

**Reply by zimbatm** [reply]
Sounds good; feel free to add me! As zowoq said you should be able to transfer your repo and also will keep commit access.

**Reply by cstrahan** [reply]
Awesome -- I'll see if I can get the rust crate up shortly! :)

**Reply by cstrahan** [reply]
Just now published the crate and sent you an owner invite :)

https://crates.io/crates/tree-sitter-nix

**Comment by zowoq** [comment, marked as answer]
Sound good, as you're already a member of nix-community you should be able to transfer the repo.

I've created a team https://github.com/orgs/nix-community/teams/tree-sitter-nix for it.

**Reply by cstrahan** [reply]
Cool, just transferred it over -- thanks!
